docs/content/developers/design-keuzes.md

```diff
@@ -121,14 +121,12 @@ De URLs in dit voorbeeld zijn uiteraard fictief.
   ```
 
 
-## Gerelateerde objecten zonder eigen resource / Groepsattributen
+## Groepsattributen
 
-Indien een (hoofd)object een gerelateerd object (of lijst van objecten) heeft,
-dat **geen** eigen resource URL nodig heeft, dan wordt het (child)object inline
-(binnen het hoofdobject) ontsloten. Deze child objecten zijn binnen RGBZ
-ook wel gedefinieerd als *groepsattributen*. Typische voorbeelden zijn
-(zaak-)kenmerken of (zaak-)eigenschappen waarin `Zaak` het hoofdobject betreft,
-en `Kenmerk` en `Eigenschap` de child objecten.
+Indien een object een groepattribuutsoort heeft (een groep van bij elkaar behorende attribuutsoorten), al dan niet herhalend d.w.z. kardinaliteit 1 of meer (een 'lijst' van waarden van de attributen van het groepattribuut), dan wordt het groepattribuut inline
+(binnen het object) ontsloten. Het krijgt dus geen eigen resource, het heeft immers geen zelfstandig bestaantsrecht. Het heeft alleen bestaansrecht als eigenschap van het object, net zoals attribuutsoorten. 
+Typische voorbeelden zijn (zaak-)kenmerken of (zaak-)eigenschappen waarin `Zaak` het hoofdobject betreft,
+en `Kenmerk` en `Eigenschap` de groepattributen.
```

Comment:
`groepattributen` hoort `groepsattributen` te zijn

Reply:
Waar staat dat? In het MIM heten het toch echt groepattribuutsoorten. Dus groepattributen.

Reply:
Op https://www.gemmaonline.nl/index.php/BRP_in_RSGB_met_Groepsattributen wordt tussen-s gebruikt, en zonder tussen-s spreekt het m.i. raar uit (heb het ook nooit zonder tussen-s gehoord). De Nederlandse spellingsregels stellen dat je dan ook de tussen-s hoort te schrijven :-)

Reply:
Helaas houden niet alle colega's zich aan het afgesproken taalgebruik. Overigens staat de groepattribuutsoort niet in het MIM. Het blijkt een alweer achterhaalde term te zijn. Het wordt inmiddels Gegevensgroeptype genoemd. Maar dus niet Gegevensgroepstype. Dus weg met die S ...

Reply:
OK, helder. Als het maar consistent is

Reply:
Zonder s, dus. Aangepast.